docs/sub-activate.md

```diff
@@ -82,26 +41,35 @@ For this step, you must have a sufficient amount of XRT tokens (minimum of 2-3 X
 
 Now you need to setup you subscription by adding the `CONTROLLER` account to it.
 
-<robo-wiki-video autoplay loop controls :videos="[{src: 'Qmd5P356UE1yDLAd4uSdq1dERbyp5gk5wpWD3iENNt2mjV', type:'mp4'}]" />
+<robo-wiki-picture src="home-assistant/sub-setup.png" />
+
+1. Go to the Robonomics dApp and navigate to the [setup a subscription page](https://robonomics.app/#/rws-setup). Navigate to the **Subscription settings** section.
 
-1. Go to the Robonomics dApp and navigate to the [setup a subscription page](https://robonomics.app/#/rws-setup). Navigate to the **GENERAL SETTINGS** section.
+2. In the `Controller's seed phrase` field press magic wand to create new `CONTROLLER` account.
 
-2. Remove the seed phrase from the `Controller's seed phrase` field and enter the `CONTROLLER` account seed phrase into.
+3. in the pop-up create password for the `CONTROLLER` account.
 
-3. Copy the `CONTROLLER` address: open the extension and click on the icon next to the account name.
+4. On the next pop-up, you will see the address of your new account and the mnemonic seed phrase. Save the mnemonic seed phrase securely because you will need it later 
```

Comment:
```suggestion
4. In the next pop-up, you will see the address of your new account and the mnemonic seed phrase. Save the mnemonic seed phrase securely because you will need it later 
```